File: PHP Word Examples/DocumentPropertiesApiDemo/getCustomDocumentProperties.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\DocumentPropertiesApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new DocumentPropertiesApi($configuration);

$fileName = "CustomDocumentProperties.docx";
$folder = "input";
$storage = null;
$password = null;
$result = $apiInstance->getCustomDocumentProperties($fileName, $password,$folder, $storage);

File: Python Word Examples/DocumentPropertiesApiDemo/getCustomDocumentProperties.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.document_properties_api.DocumentPropertiesApi(configuration)

file_name = 'CustomDocumentProperties.docx'
folder = 'input'
custom_properties = api.get_custom_document_properties(file_name, folder=folder)